### PR TITLE
About: use new ListItem

### PR DESCRIPTION
**Pull request by mardy**
Avoid using deprecated ListItems in the About plugin. This partially addresses https://github.com/ubports/ubuntu-ui-toolkit/issues/66

### CODE REVIEW

**Review by mariogrip — DISMISSED** [review]
LGTM :+1:

File: plugins/about/DevMode.qml [hunk 1]
```diff
@@ -2,8 +2,7 @@
  * This file is part of system-settings
  *
  * Copyright (C) 2014 Canonical Ltd.
- *
- * Contact: Oliver Grawert <ogra@canonical.com>
+ * Copyright (C) 2020 Ubports Foundation <developers@ubports.com>
```
Comment (UniversalSuperBox) [inline plugins/about/DevMode.qml]: Exactly speaking, it is capitalized UBports

**Review by UniversalSuperBox — CHANGES_REQUESTED** [review]

File: plugins/about/DevMode.qml (hunk 1 above)
Comment (mardy) [reply plugins/about/DevMode.qml]: Done!

**Comment by mardy** [comment]
Anyone knows what's the issue with the CI?

File: plugins/about/DevMode.qml [hunk 2]
```diff
@@ -2,8 +2,7 @@
  * This file is part of system-settings
  *
  * Copyright (C) 2014 Canonical Ltd.
- *
- * Contact: Oliver Grawert <ogra@canonical.com>
+ * Copyright (C) 2020 UBports Foundation <developers@ubports.com>
```
Comment (UniversalSuperBox) [inline plugins/about/DevMode.qml:5]: This e-mail does not exist. Were you expecting something different?

Comment (mardy) [reply plugins/about/DevMode.qml:5]: I was told to use this e-mail, long ago. So, what copyright line should we use?

**Review by UniversalSuperBox — APPROVED** [review]

**Pull request merged**